rpc/api.proto

```diff
@@ -459,3 +476,93 @@ message DecodeSeedRequest {
 message DecodeSeedResponse {
 	bytes decoded_seed = 1;
 }
+
+message StartTicketPurchaseRequest {
+	bytes passphrase = 1;
+	string account_name           = 2;
```

Comment:
none of the other field positions are aligned, don't do it here.

they're not even aligned correctly.